src/pages/Background/index.js

```diff
@@ -1,2 +1,78 @@
+//Even while development.. extension need to be reloaded for changes here to come into effect
+import { closest, distance } from 'fastest-levenshtein';
+import { InitialUserSetup } from './modules/InitialUserSetup';
+
 console.log('This is the background page.');
-console.log('Put the background scripts here.');
+
+
+//When extension is installed first it will popup a login and consent approval window
+chrome.runtime.onInstalled.addListener(() => {
+    chrome.storage.local.set({ 'userCreds': {} })
```

Comment:
Is it safe to store users sheet creds in `localStorage`. Other apps would be able to access this right?

We should probably encrypt it & store it in the cookies with the strictest security enforcements in place I believe.

Reply:
yes, i'll look into this